### PR TITLE
[docs] program examples

### PR DESCRIPTION
**Pull request by Woody4618**
This guide should highlight the solana program examples so that people know that they exist and has a list of all the examples with a short description and a link to the example.

### CODE REVIEW

File: docs/programs/examples.md
Comment (nickfrosty) [inline docs/programs/examples.md:69]:
```suggestion
| **[Transfer SOL](https://github.com/solana-developers/program-examples/tree/main/basics/transfer-sol)**                           | Different methods of transferring SOL for system accounts and PDAs.                               | Native, Anchor, Solang, Seahorse            |
```

File: docs/programs/examples.md
Comment (nickfrosty) [inline docs/programs/examples.md:81]:
```suggestion
| **[cutils](https://github.com/solana-developers/program-examples/tree/main/compression/cutils)**                | A suite utils to for example mint and verify cNFTs in a program.                    | Anchor   |
```

File: docs/programs/examples.md
Comment (nickfrosty) [inline docs/programs/examples.md:103]:
```suggestion
| **[Token Swap](https://github.com/solana-developers/program-examples/tree/main/tokens/token-swap)**                 | Extensive example that shows you how to build a AMM (automated market maker) pool for SPL tokens. | Anchor                           |
```

File: docs/programs/examples.md
Comment (nickfrosty) [inline docs/programs/examples.md:34]: Let's put the link here, and then the repo path where the link was:

```suggestion
The
"[Solana Program Examples](https://github.com/solana-developers/program-examples)"
repository on GitHub offers several subfolders, each containing code examples
for different Solana programming paradigms and languages, designed to help
developers learn and experiment with Solana blockchain development.
```

File: docs/programs/examples.md
Comment (nickfrosty) [inline docs/programs/examples.md:38]:
```suggestion
You can find the examples in the `solana-developers/program-examples` together
with README files that explain you how to run the different examples. Most
```

File: docs/programs/examples.md
Comment (nickfrosty) [inline docs/programs/examples.md:42]: let's not put solang in here...

```suggestion
Most examples are self-contained and are available in Native rust,
[Anchor](https://www.anchor-lang.com/docs/installation) (rust), and
[Seahorse](https://seahorse-lang.org/) (python).
```

File: docs/programs/examples.md
Comment (nickfrosty) [inline docs/programs/examples.md:44]: let's link directly to it!

```suggestion
It also contains a list of examples that we would love to
[see as contributions](https://github.com/solana-developers/program-examples?tab=readme-ov-file#examples-wed-love-to-see).
```

File: docs/programs/examples.md
Comment (nickfrosty) [inline docs/programs/examples.md:46]:
```suggestion
Within the repo you will find the following subfolder, each with assorted
example programs within them:

- [`basics`](#basics)
- [`compression`](#compression)
- [`oracles`](#oracles)
- [`tokens`](#tokens)
```

**Review by nickfrosty — CHANGES_REQUESTED** [review]
I left some feedback

Can you also:
- remove the bold from all the table links?
- lets also go ahead and delete the "Language Specific" section at the end since the tables cover which language is each example has

**Comment by jacobcreech** [comment]
Could this be a better replacement page for https://solana.com/docs/programs/examples instead of a separate guide?

**Comment by Woody4618** [comment]
> Could this be a better replacement page for https://solana.com/docs/programs/examples instead of a separate guide?

@jacobcreech yes it is a replacement for the program examples page. Nick had the same idea and i moved it already.

File: docs/programs/examples.md
Comment (mikemaccana) [inline docs/programs/examples.md:41]: Just a suggestion that since Seahorse is looking for a permanent maintainer, we hold off to avoid steering people in this direction. We can refocus on Seahorse once we have a maintainer and it's up to date with Solana CLI releases.

Comment (mikemaccana) [inline docs/programs/examples.md:41]:
```suggestion
examples are self-contained and are available in native Rust (ie, with no framework) and using [Anchor](https://www.anchor-lang.com/docs/installation), and it also contains a list of examples
```

**Review by mikemaccana — APPROVED** [review]
All good, just a question regarding Seahorse.

File: docs/programs/examples.md
Comment (nickfrosty) [reply docs/programs/examples.md:41]: Anvit took over being the maintainer a few months ago. Is he not actually maintaining it?

**Review by nickfrosty — APPROVED** [review]
I just pushed some last minute typo fixes, but the rest looks good. Thanks for updating all of this!

**Pull request merged**
